bin/create-embed-test-post.php

```diff
@@ -35,10 +209,63 @@ function amp_test_prepare_image_attachments( $data ) {
 	) );
 	$data['ids'] = wp_list_pluck( $attachments, 'ID' );
 
-	// @todo Add some attachments if count( $data['ids'] ) < 5.
+	$image_count_needed = 5;
```

Comment:
I can't think of a case where this would be needed as a fallback rather than directly getting the images like we do for the other shortcodes (please advise if I am missing something). [This commit](https://github.com/Automattic/amp-wp/pull/829/commits/b8ba8369d07234c753f5fe7a801a09708b4fc9ed) simplifies this.